src/voodoo/conversations/VoodooConversations.ts

```diff
@@ -46,37 +80,52 @@ export default class VoodooConversations {
   }
 
   // Must be locked by convoMutex
+  // TODO: this method currently looks up the other party's multibundle for NxN fanout
+  // we may want to carry this information in the invite itself otherwise there could be
+  // a mismatch on the initial set of devices and the set we resolve in this method
   async processInvite(
```

Comment:
This function is currently incomplete but will addressed in the next PR. New logic will need to be:
- On invite, check if a new session needs to be added to an existing conversation
- If brand new conversation, do a `multibundle` lookup and initiate OneToOneSession's with each of those instances
- If new self-conversation, initiate `multibundle` lookup for both senderAddress and self, then initiate OneToOneSessions with each of those entities if not already in the MultiSession object of the VoodooConversation